homeassistant/components/nuimo_controller.py

```diff
@@ -0,0 +1,186 @@
+"""
+Component that connects to a Nuimo device over Bluetooth LE.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/nuimo_controller/
+"""
+import logging
+import threading
+import time
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util as util
+from homeassistant.const import (CONF_MAC, CONF_NAME, EVENT_HOMEASSISTANT_STOP)
+
+REQUIREMENTS = [
+    '--only-binary=all '  # avoid compilation of gattlib
+    'git+https://github.com/getSenic/nuimo-linux-python'
+    '#nuimo==1.0.0&subdirectory=nuimo']
```

Comment:
So this will not working with our check if a package exists, causing it to be re-installed on every launch of the platform as it will feed this whole string to `urlparse.parse`, which will choke.